src/app/routes/api/v3/forms/__tests__/public-forms.verification.routes.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { StatusCodes } from 'http-status-codes'
+import session, { Session } from 'supertest-session'
+
+import { BasicField } from 'src/types'
+
+import { setupApp } from 'tests/integration/helpers/express-setup'
+import { generateDefaultField } from 'tests/unit/backend/helpers/generate-form-data'
+import dbHandler from 'tests/unit/backend/helpers/jest-db'
+
+import { PublicFormsVerificationRouter } from '../public-forms.verification.routes'
+
+const verificationApp = setupApp('/forms', PublicFormsVerificationRouter)
+
+jest.mock('nodemailer', () => ({
+  createTransport: jest.fn().mockReturnValue({
+    sendMail: jest.fn().mockResolvedValue(true),
+  }),
+}))
+// Default Twilio export is a function
+jest.mock('twilio', () => () => ({
+  messages: {
+    create: jest.fn().mockResolvedValue({
+      sid: 'mockSid',
+    }),
+  },
+}))
+
+describe('public-forms.verification.routes', () => {
+  let mockEmptyFormId: string
+  let mockVerifiableFormId: string
+  let request: Session
+
+  beforeAll(async () => await dbHandler.connect())
+
+  beforeEach(async () => {
+    request = session(verificationApp)
+    jest.clearAllMocks()
+    await dbHandler.clearDatabase()
+
+    // Form without verifiable fields
+    const { form: emptyForm } = await dbHandler.insertEmailForm()
```

Comment:
may want to inline this since this is only used in a single test. Same for the verifiable form

Reply:
i didn't do this cos it'll be reused for the related endpoints in the issues @__@ if it's an issue, i can change it!